omnigibson/prims/geom_prim.py

```diff
@@ -157,7 +157,8 @@ def _post_load(self):
             self._mesh_collision_api = UsdPhysics.MeshCollisionAPI(self._prim) if \
                 self._prim.HasAPI(UsdPhysics.MeshCollisionAPI) else UsdPhysics.MeshCollisionAPI.Apply(self._prim)
             # Set the approximation to be convex hull by default
-            self.set_collision_approximation(approximation_type="convexHull")
+            if self.get_collision_approximation() != "boundingSphere":
+                self.set_collision_approximation(approximation_type="convexHull")
```

Comment:
This is code Josiah included because for Fetch's wheels that were boundingSpheres, they were being approximated as ConvexHulls which were causing issues. The wheels were jagged and not making smooth contact with the floor. @cremebrule may know what the best thing to do here is.

Reply:
yeah great points. Agreed, it's a bit hacky to hardcode a very general fix (special-case boundingSpheres) for the sake of a very specific use-case (Fetch wheels).

I think the best would be to keep it object-centric -- i.e.: implement a fix directly in Fetch (e.g.: Fetch in turn overrides its own wheel meshes to be boundingSphere again, after the geom_prim by default sets itself to be convexHull)

Reply:
Ideally during asset USD generation, apply collision approximation (convex hull, bbox, or sphere, whichever fits), and then hopefully this approximation is GPU compatible. 

Temporary: add the hack in fetch.py